src/workspacer.Sound/SoundPluginConfig.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace workspacer.Sound
+{
+    public class SoundPluginConfig
+    {
+        public bool BindDefaultPlaybackKeybinds { get; set; }
```

Comment:
In this case, default = `false`, which should result in the behaviour you are complaining about being fixed... But then it can be overridden if the user wants to. Right? If so, that's good.

Reply:
Well I suspect most workspaces users would use this only for the widget and not the actual control and bindings. That's why I left this default false so no double actions would happen on the media sound controls